Rover/fence.cpp

```diff
@@ -67,6 +67,11 @@ void Rover::fence_check()
                         set_mode(mode_hold, ModeReason::FENCE_BREACHED);
                     }
                     break;
+                case FailsafeAction::Loiter_Hold:
```

Comment:
I think you need the docs for this one up in AC_Fence too.

Reply:
https://github.com/ArduPilot/ardupilot/blob/7c0f01bd8d4c335a3fb962cfce925640cc1d4307/libraries/AC_Fence/AC_Fence.cpp#L70

Reply:
thanks! that was missing "terminate" too